hw/bsp/msp_exp430f5529lp/board.mk

```diff
@@ -0,0 +1,32 @@
+CFLAGS += \
+  -D__MSP430F5529__ \
+  -DCFG_TUSB_MCU=OPT_MCU_MSP430x5xx \
+	-DCFG_EXAMPLE_MSC_READONLY \
+	-DCFG_TUD_ENDPOINT0_SIZE=8
+
+#-mmcu=msp430f5529
+
+# All source paths should be relative to the top level.
+LD_FILE = hw/mcu/ti/msp430/msp430-gcc-support-files/include/msp430f5529.ld
+LDINC += $(TOP)/hw/mcu/ti/msp430/msp430-gcc-support-files/include
+
+INC += $(TOP)/hw/mcu/ti/msp430/msp430-gcc-support-files/include
+
+# For TinyUSB port source
+VENDOR = ti
+CHIP_FAMILY = msp430x5xx
+
+# Allow user to use mspdebug if set on make command line.
+ifdef MSPDEBUG
+# flash target using mspdebug.
+flash: $(BUILD)/$(BOARD)-firmware.elf
```

Comment:
aha, I thought people would like to use MSP430Flasher more since it is from TI. But not at all, maybe just define it with `flash-mspdebug` target to skip entering that via command line.

Reply:
Oh right, the reason I didn't do `flash-mspdebug` is that the `which` line spits out a bunch of errors if `MSP430Flasher` is _not_ on your path. I'd like to avoid noise from that :P.

Reply:
Actually, let's gate the `LD_LIBRARY_PATH` line to not run on Windows instead... that should work fine.

Reply:
yeah, let's do that, much nicer, I still don't know why it could pickup the libmsp430.so when it is located in the same folder with MSP430Flasher in Linux :thinking: but anyway !!